index.html

```diff
@@ -333,12 +333,240 @@
+          limitations. Examples are set-top boxes, smart TVs or networked
+          speakers capable of rendering only certain formats of video and/or audio.
+          We say that such a device is a <dfn>compatible remote playback device</dfn> for a
+          <a>media element source</a> if the <a>user agent</a> can
+          reasonably guarantee that the remote playback of the media speficied by the source
+          on that device will succeed.
+        </p>
+      </section>
+      <section>
+        <h4>
+          Getting the <a>remote playback devices</a> availability information
+        </h4>
+        <p>
+          When the <code><dfn for="RemotePlayback">getAvailability</dfn>()</code> method is
+          called, the user agent MUST run the following steps:
+        </p>
+        <dl>
+          <dt>
+            Input
+          </dt>
+          <dd>
+            <var>remote media source</var>, the <a>media element source</a> preferred for remote playback
+          </dd>
+          <dt>
+            Output
+          </dt>
+          <dd>
+            <var>P</var>, a <a>Promise</a>
+          </dd>
+        </dl>
+        <ol>
+          <li>Let <var>P</var> be a new <a>Promise</a>.
+          </li>
+          <li>Return <var>P</var>, but continue running these steps <a>in
+          parallel</a>.
+          </li>
+          <li>If the user agent is unable to <a>monitor the list of available
+          remote playback devices</a> for the entire lifetime of the browsing context
+          (for instance, because the user has disabled this feature), then:
+            <ol>
+              <li>
+                <a>Resolve</a> <var>P</var> with a new
+                <code>RemotePlaybackAvailability</code> object with its
+                <code>value</code> property set to <code>false</code>.
+              </li>
+              <li>Abort all the remaining steps.
+              </li>
+            </ol>
+          </li>
+          <li>If the user agent is unable to continuously <a>monitor the list
+          of available remote playback devices</a> but can do it for a short period of time
+          when initiating remote playback, then:
+            <ol>
+              <li>
+                <a>Reject</a> <var>P</var> with a <a>NotSupportedError</a>
+                exception.
+              </li>
+              <li>Abort all the remaining steps.
+              </li>
+            </ol>
+          </li>
+          <li>
+            If there exists a tuple <em>(<var>A</var>,
+            <var>remote media source</var>)</em> in the <a>set of availability
+            objects</a>, then:
+            <ol>
+              <li>
+                <a>Resolve</a> <var>P</var> with <var>A</var>.
+              </li>
+              <li>
+                Abort all the remaining steps.
+              </li>
+            </ol>
+          </li>
+          <li>
+            Let <var>A</var> be a new <code>RemotePlaybackAvailability</code>
```

Comment:
Done. I liked short names that stand out on its own though because of a capital letter even if CSS fails.